packages/squiggle-lang/tsconfig.json

```diff
@@ -15,6 +16,7 @@
     "target": "ES6",
     "esModuleInterop": true
   },
+  "files": ["./__tests__/malaria_consortium.json"],
```

Comment:
Is this necessary? The json file is already in `__tests__` which is included below.

Reply:
Apparently it is? I get an error without this line when I try to compile.